### PR TITLE
Iron-collapse removed, div with id content css transition.

### LINKED ISSUE #46: replace iron-collapse
iron-collapse should also be replaced with a simple CSS transition and a transition-end event.

### PR DESCRIPTION
**Pull request by programmer4web**
Replaced iron-collapse in cosmoz-tab.
Replaced iron-collapse in cosmoz-tab-card.
Tests updated.
es6 updates for accordion.html tests.
fixes #46 
More tests to keep up with the code coverage decrease. Done.

### CODE REVIEW

**Comment by megheaiulian** [comment]
Needs more tests to keep up with the code coverage decrease.

File: cosmoz-tab.html [hunk 1]
```diff
@@ -125,8 +123,15 @@
 				@apply --layout-wrap;
 			}
 
-			:host([has-cards][accordion])  #content {
-				display: block;
+			:host(:not([has-cards])[accordion]) #content {
+				transition-duration: var(--cosmoz-tab-collapse-transition-duration, 300ms);
```
Comment (megheaiulian) [inline cosmoz-tab.html]: `--cosmoz-tab-collapse-transition-duration` should be `--cosmoz-tab-transition-duration` similar to the variable for tab-card

**Comment by programmer4web** [comment]
@megheaiulian This is ready for review. (more tests added to keep up with the code coverage)

**Comment by megheaiulian** [comment]
@nomego On desktop (non accordion) iron-collpase wasn't used at all but still handled and triggered some  resize events. Since it's not needed we just do the transitions with css on mobile.

**Comment by nomego** [comment]
Does this still trigger "tab activated" when expanding "tab as card" on mobile?

**Comment by megheaiulian** [comment]
@nomego This should work exactly as before. The transition part was already totally separated from the activated event. Normally it just changed the `isSelected` (with reflectToAttribute) property and iron-collapse used that for a transition. Now we use `isSelected` to trigger a CSS transition.

**Pull request merged**